homeassistant/components/__init__.py

```diff
@@ -168,7 +169,7 @@ def async_handle_turn_service(service):
     def async_handle_core_service(call):
         """Service handler for handling core services."""
         if call.service == SERVICE_HOMEASSISTANT_STOP:
-            hass.async_create_task(hass.async_stop())
+            async_call_later(hass, 0, hass.async_stop())
```

Comment:
Home Assistant is a layered system. It's weird to change behavior in the core because a few layers on top is reporting the wrong result.

The only thing that `async_call_later` does is to delay the call between very little time to up to a second (as we don't know when next time changed event will fire). We're actually working on removing time changed event and replace it with `hass.loop.call_soon`.

What is the HTTP message that is causing the error and could we update `<ha-call-service-button>` on the frontend to catch it correctly instead?